### ISSUE TITLE
README: we recommend using the once_cell crate to ensure that regular expressions are compiled

### ISSUE THREAD

**Issue by kzhui125** [rust-lang/regex#1217]
LazyLock is stabilized in 1.80, maybe it's recommend than once_cell crate

**Comment by BurntSushi** [comment]
Yes. But let's wait a release cycle or two. `LazyLock` was _just_ stabilized.

**Comment by nyurik** [comment]
@BurntSushi, I think we are safe - it has been 5 releases now :)  Thx @noritada for creating a PR!
